signatures/helpers/arguments_helpers.go

```diff
@@ -44,7 +44,7 @@ func GetTraceeArgumentByName(event trace.Event, argName string, opts GetArgOps)
 	return trace.Argument{}, fmt.Errorf("argument %s not found", argName)
 }
 
-// GetTraceeStringArgumentByName gets the argument matching the "argName" given from the event "argv" field, casted as string.
+// GetTraceeStringArgumentByName gets the argument matching the "argName" given from the event "args" field, casted as string.
```

Comment:
```suggestion
// GetTraceeStringArgumentByName retrieves the argument from the event's "Args" field 
// that matches the specified "argName". The argument value is returned cast as a string.
```

@ShohamBit WDYT? If you like it, replicate the pattern below.